package.json

```diff
@@ -52,9 +52,10 @@
   },
   "dependencies": {
     "lodash.unescape": "4.0.0",
-    "object-assign": "^4.0.1"
+    "object-assign": "^4.0.1",
+    "semver": "^4.3.6"
```

Comment:
Sure!

Interesting message on running `npm install` here, btw.

`npm WARN The package semver is included as both a dev and production dependency.`